Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
```

Comment:
### [audited](https://github.com/collectiveidea/audited)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.4.2 → 5.4.3

[_~~change-log~~, [source-code](https://github.com/collectiveidea/audited), [gem-diff](https://my.diffend.io/gems/audited/5.4.2/5.4.3)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/collectiveidea/audited/compare/v5.4.2...v5.4.3).

These are the first 10 commits:
- (73c202c) [Add config ignored_default_callbacks](https://github.com/collectiveidea/audited/commit/73c202c5da2817d8c1bf433f99dddadd853f17dd)
- (ab22a3f) [Ignore readonly columns in audit](https://github.com/collectiveidea/audited/commit/ab22a3f165a2927f4f21348d050dd56aad7c0b97)
- (e361f02) [Merge pull request #692 from sriddbs/ignore-readonly-attrs](https://github.com/collectiveidea/audited/commit/e361f025eb49badf7aeaec4c8749ba85b1d44df6)
- (e163219) [Robustify Rails version checks](https://github.com/collectiveidea/audited/commit/e1632193460eef92b290938d3f94dbc315e8cd43)
- (4606723) [Merge pull request #689 from blaet/robustify-rails-version-c…](https://github.com/collectiveidea/audited/commit/4606723f0d43c4d583327b28366b2dbd227cf4cf)
- (cd0ac65) [Update comment to match code](https://github.com/collectiveidea/audited/commit/cd0ac65864af82a1e707ca9063296cb036cd58f5)
- (a609166) [Add note about touch audits to README](https://github.com/collectiveidea/audited/commit/a609166101383c65a08c209655a54c4eec6923f8)
- (00a1d82) [Merge pull request #679 from JDrizzy/ignore-callbacks](https://github.com/collectiveidea/audited/commit/00a1d82bb350122b6af0652c395509682d5fa98d)
- (8368291) [Update Changelog](https://github.com/collectiveidea/audited/commit/836829116376c2aa6054f59f915d430a1243f9dc)
- (fa53258) [Bump version](https://github.com/collectiveidea/audited/commit/fa53258b019624359fa9ef34fe8853dc447f58b3)


</details>